source/testnet/guides/become-baker.rst

```diff
@@ -135,6 +110,19 @@ concept of an *epoch* as a unit of time that defines a period in which the set
 of current bakers and stakes are fixed. Epochs have a time duration fixed at the
 Genesis block. In the testnet, epochs have a duration of **1 hour**.
 
+When performing changes, once the transaction that performs the change is
+finalized, this change can be in two different states:
```

Comment:
```suggestion
Changes to the baker state can be in one of the following states:
```

Reply:
But should this paragraph be in a different section? I realize that it's probably under `Epochs and slots` because times are often expressed in terms of epochs and slots. However, I think at this point it might look out of context. Maybe we could move this further down where it becomes relevant?

Reply:
This is entirely optional, especially because we have little time left and a lot of work to do, but I wonder if it could simplify/clarify/remove the amount of writing if we included a little table that shows when the different baker events become visible by querying the node vs. when they actually take effect. Something on the lines of the [confluence table](https://concordium.atlassian.net/wiki/spaces/GEN/pages/435814401/Manual+Testing+FAQ#When-do-on-chain-events-happen?).

Reply:
This section has been removed and some tables have been added. Hopefully it is clearer now.